### DISCUSSION TITLE
I'm not sure if I should open a bug, but any ideas on why logos are wrong on the popup?

### DISCUSSION THREAD

**Discussion by gabrieljim** [family/connectkit#124] [Q&A] [answered]
At first, it looks like this:
![image](https://user-images.githubusercontent.com/21229805/219534288-f3dbf791-d5b1-4d48-97f0-09f1dc73f0e9.png)

But after less than a second:
![image](https://user-images.githubusercontent.com/21229805/219534344-2799aba3-7696-4185-abee-3eff0afd7917.png)

**Comment by lochie** [comment, marked as answer]
looks like there might be a css conflict somewhere, @gabrieljim do you have a demo i can check out?

**Reply by gabrieljim** [reply]
I'm sorry, it's a company thing so I can't quite share it, but thanks for pointing me in a direction! I'll dig in for css conflicts

**Reply by gabrieljim** [reply]
Now I'm glad I didn't open a bug hahah, I'm using a extension called "Dark Reader" which was turning on by itself so it made some colors not be there, apologies for the dumb question!

**Reply by lochie** [reply]
no sweat! always happy to help ☺️
